### PR TITLE
Feature: increased code coverage

### PR DESCRIPTION
**Pull request by wakr**

### CODE REVIEW

File: src/test/java/hy/tmc/cli/frontend/communication/commands/ReplyToPingTest.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+public class ReplyToPingTest {
+
+    @Test
+    public void pingAnswersAsPong() {
+        FrontendStub frontStub = new FrontendStub();
+        ReplyToPing ping = new ReplyToPing(frontStub);
+        ping.functionality();
+        assertEquals("pong", frontStub.getMostRecentLine());
+
```
Comment (samutamm) [inline src/test/java/hy/tmc/cli/frontend/communication/commands/ReplyToPingTest.java]: remove empty line

File: testResources/failingExercise/viikko1/Viikko1_001.Nimi/build/built-jar.properties [hunk 2]
```diff
@@ -1,4 +1,4 @@
-#Mon, 08 Jun 2015 09:27:23 +0300
+#Mon, 08 Jun 2015 10:04:28 +0300
 
 
-/home/ad/fshome6/u6/x/xtoxtox/Linux/tmc-cli/testResources/failingExercise/viikko1/Viikko1_001.Nimi=
+/media/ExternalSD/Other/OhTu/tmc-cli/testResources/failingExercise/viikko1/Viikko1_001.Nimi=
```
Comment (jamox) [inline testResources/failingExercise/viikko1/Viikko1_001.Nimi/build/built-jar.properties]: Why these files even exist?

File: testResources/failingExercise/viikko1/Viikko1_001.Nimi/build_log.txt [hunk 3]
```diff
@@ -42,12 +42,12 @@
...
-   [delete] Deleting: /home/ad/fshome6/u6/x/xtoxtox/Linux/tmc-cli/testResources/failingExercise/viikko1/Viikko1_001.Nimi/build/built-jar.properties
+   [delete] Deleting: /media/ExternalSD/Other/OhTu/tmc-cli/testResources/failingExercise/viikko1/Viikko1_001.Nimi/build/built-jar.properties
 
 deps-jar:
 
 -warn-already-built-jar:
-[propertyfile] Updating property file: /home/ad/fshome6/u6/x/xtoxtox/Linux/tmc-cli/testResources/failingExercise/viikko1/Viikko1_001.Nimi/build/built-jar.properties
+[propertyfile] Updating property file: /media/ExternalSD/Other/OhTu/tmc-cli/testResources/failingExercise/viikko1/Viikko1_001.Nimi/build/built-jar.properties
 
 -check-automatic-build:
```
Comment (jamox) [inline testResources/failingExercise/viikko1/Viikko1_001.Nimi/build_log.txt]: Why these files even exist?

File: testResources/successExercise/viikko1/Viikko1_001.Nimi/build/built-jar.properties [hunk 4]
```diff
@@ -1,4 +1,4 @@
-#Mon, 08 Jun 2015 09:27:24 +0300
+#Mon, 08 Jun 2015 10:04:29 +0300
 
 
-/home/ad/fshome6/u6/x/xtoxtox/Linux/tmc-cli/testResources/successExercise/viikko1/Viikko1_001.Nimi=
+/media/ExternalSD/Other/OhTu/tmc-cli/testResources/successExercise/viikko1/Viikko1_001.Nimi=
```
Comment (jamox) [inline testResources/successExercise/viikko1/Viikko1_001.Nimi/build/built-jar.properties]: Why these files even exist?

File: testResources/successExercise/viikko1/Viikko1_001.Nimi/build_log.txt [hunk 5]
```diff
@@ -42,12 +42,12 @@
...
-   [delete] Deleting: /home/ad/fshome6/u6/x/xtoxtox/Linux/tmc-cli/testResources/successExercise/viikko1/Viikko1_001.Nimi/build/built-jar.properties
+   [delete] Deleting: /media/ExternalSD/Other/OhTu/tmc-cli/testResources/successExercise/viikko1/Viikko1_001.Nimi/build/built-jar.properties
 
 deps-jar:
 
 -warn-already-built-jar:
-[propertyfile] Updating property file: /home/ad/fshome6/u6/x/xtoxtox/Linux/tmc-cli/testResources/successExercise/viikko1/Viikko1_001.Nimi/build/built-jar.properties
+[propertyfile] Updating property file: /media/ExternalSD/Other/OhTu/tmc-cli/testResources/successExercise/viikko1/Viikko1_001.Nimi/build/built-jar.properties
 
 -check-automatic-build:
```
Comment (jamox) [inline testResources/successExercise/viikko1/Viikko1_001.Nimi/build_log.txt]: Why these files even exist?

**Pull request merged**